stylesheets/mapLocations.css

```diff
@@ -0,0 +1,245 @@
+/* This file captures styling information for placement and overlay of the map. */
+
+/* Fall */
+.Fall-pos0 {
```

Comment:
I've been thinking on this a bit. I think I like how you've set it up - you've kept "data.js" limited to non-view specific concerns as much as possible. However, it does lead to: 1. you need to modify multiple places if adding a new map (data.js and this CSS file), and 2. this CSS file has a lot of duplicate boilerplate just to convey a bunch of positional data. That's kinda just how CSS is, at some level, I guess.

I wonder if at some point maybe we want like a uiData.js file, which has view-specific data for the various entities in the system? And that would include all this positional data for maps, image file paths, clearing colors, etc, which our view code then reads and applies dynamically. That eliminates a bunch of CSS boilerplate, but it does still require a person to modify multiple places when adding a new entity. But just editing two data files may be easier than editing data and CSS?

Reply:
Yea I think this could use a refactor. 

I initially wanted to programmatically generate the CSS classes when the map is being selected. The pixel offsets could just be encoded in an array that we walk through. Your uiData.js idea is similar to this, but I assume you intend to just create all of them during file-load?

Reply:
No, I was thinking the same as you. I guess technically generating classes should happen in a compile step, but that's totally overkill.